apps/scan/backend/src/app.ts

```diff
@@ -89,6 +89,17 @@ function buildApi(
       const [usbDrive] = await usb.getUsbDrives();
       assert(usbDrive?.mountPoint !== undefined, 'No USB drive mounted');
 
+      const authStatus = await auth.getAuthStatus(
+        constructAuthMachineState(workspace)
+      );
+      if (authStatus.status !== 'logged_in') {
+        return err('auth_required_before_ballot_package_load');
+      }
+
+      if (authStatus.user.role !== 'election_manager') {
```

Comment:
The frontend should prevent this from happening but we enforce it here for redundancy